packages/pages/src/components/image/image.tsx

```diff
@@ -70,47 +73,89 @@ export const Image = ({
     .map((w) => `${getImageUrl(imgUUID, w, (imgHeight / imgWidth) * w)} ${w}w`)
     .join(", ");
 
-  const onError = () => {
-    setImgStatus(imgLoadingStatus.ERROR);
-    console.warn(`Invalid image src: ${src}.`);
-  };
-
   return (
     <>
-      {placeholder != null &&
-        imgStatus != imgLoadingStatus.LOADED &&
-        placeholder}
-      {imgStatus != imgLoadingStatus.ERROR && (
+      {!imgLoaded && placeholder != null && placeholder}
+      {src && (
         <img
           ref={imgRef}
           style={imgStyle}
           src={src}
           className={className}
-          width={width}
-          height={height}
+          width={absWidth}
+          height={absHeight}
           srcSet={srcSet}
           loading={"lazy"}
-          onLoad={() => setImgStatus(imgLoadingStatus.LOADED)}
-          onError={() => onError()}
+          onLoad={() => setImgLoaded(true)}
           {...imgOverrides}
         />
       )}
     </>
   );
 };
 
+// Checks if required props are passed in for the specified layout, if not, log a warning.
+export const validateRequiredProps = (
+  layout: ImageLayout,
+  imgWidth: number,
+  imgHeight: number,
+  width?: number,
+  height?: number,
+  aspectRatio?: number
+) => {
+  if (imgWidth < 0) {
+    console.warn(`Invalid image width: ${imgWidth}.`);
+  }
+
+  if (imgHeight < 0) {
+    console.warn(`Invalid image height: ${imgHeight}.`);
+  }
+
+  if (layout == ImageLayout.FIXED) {
+    if (!width && !height) {
```

Comment:
Is height actually required? When I was testing earlier a fixed image with just a width seemed fine.

Reply:
we need at least a width or a height to render

Reply:
Oh yea, ha, misread. 😓